Gemfile.lock

```diff
@@ -48,32 +47,33 @@ GEM
     method_source (0.9.2)
     mini_portile2 (2.4.0)
     minitest (5.11.3)
-    money (6.13.2)
+    money (6.13.4)
       i18n (>= 0.6.4, <= 2)
-    nokogiri (1.10.1)
+    nokogiri (1.10.4)
       mini_portile2 (~> 2.4.0)
     pagerduty (2.1.2)
       json (>= 1.7.7)
-    rack (2.0.6)
+    rack (2.0.7)
     rack-test (1.1.0)
       rack (>= 1.0, < 3)
     rails-dom-testing (2.0.3)
       activesupport (>= 4.2.0)
       nokogiri (>= 1.6)
-    rails-html-sanitizer (1.0.4)
+    rails-html-sanitizer (1.2.0)
       loofah (~> 2.2, >= 2.2.2)
-    railties (5.2.2.1)
-      actionpack (= 5.2.2.1)
-      activesupport (= 5.2.2.1)
+    railties (6.0.0)
+      actionpack (= 6.0.0)
+      activesupport (= 6.0.0)
       method_source
       rake (>= 0.8.7)
-      thor (>= 0.19.0, < 2.0)
-    rake (12.3.2)
+      thor (>= 0.20.3, < 2.0)
+    rake (12.3.3)
```

Comment:
### [rake](https://github.com/ruby/rake)

**Patch** version upgrade :chart_with_upwards_trend::small_blue_diamond: 12.3.2 → 12.3.3

[_~~change-log~~, [source-code](https://github.com/ruby/rake)_]





<details>
<summary>Commits</summary>

A change of **25** commits. See the full changes on [the compare page](https://github.com/ruby/rake/compare/v12.3.2...v12.3.3).

These are the first 10 commits:
- (3d5a5be) [Add ruby 2.6.0 to .travis.yml](https://github.com/ruby/rake/commit/3d5a5be09038c160fa6ec9c3186a5c8a24d7d8d8)
- (d21463c) [Merge pull request #300 from ruby/colby/add-ruby-2.6](https://github.com/ruby/rake/commit/d21463c8d552327c043c670288fa0e01f601b2ef)
- (799d847) [fix outstanding rubocop warnings](https://github.com/ruby/rake/commit/799d84787fd4064f005a8383391b8f3a402007fc)
- (9d9b431) [Merge pull request #301 from ruby/colby/update-rubocop](https://github.com/ruby/rake/commit/9d9b4315a12d705b4fcd42c2c25735ef17ec289e)
- (d28957d) [Use the application's name in error message if a task is not…](https://github.com/ruby/rake/commit/d28957d64ae88823200049f8ae3667eb631bdfcc)
- (67ced2f) [Merge pull request #303 from tmatilai/app-name-in-error](https://github.com/ruby/rake/commit/67ced2fab148db0ada6b0d52d321f5cc86e22960)
- (7b75d7a) [Use Ruby 2.6.1](https://github.com/ruby/rake/commit/7b75d7a084c6408759d745db270550b8d14d02cf)
- (91e4b80) [Merge pull request #305 from aycabta/use-2.6.1](https://github.com/ruby/rake/commit/91e4b80784b9964e9b4d1cc9e79e5def38f9c53b)
- (aec6e97) [Set up CI with Azure Pipelines](https://github.com/ruby/rake/commit/aec6e976a11728ec2fc78946f308b28d9b2522a3)
- (48a5f2e) [Applied matrix build for the multiple platforms.](https://github.com/ruby/rake/commit/48a5f2e9b888dd5eb1aa9c7aa624c4191a4c2bae)


</details>